docs/aws/audit/shieldmonitoring/rules/aws_shield_drt_access_remediation.mdx

```diff
@@ -1,6 +1,252 @@
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console and open the AWS Shield console at https://console.aws.amazon.com/shield/.
+   
+2. In the navigation pane, choose "Settings". 
+
+3. Under "AWS Shield Response Team access", check if the "Allow access" option is enabled. If it is enabled, it means the Shield Response Team (SRT) has access to your AWS account.
+
+4. For further verification, you can also check the IAM policies and roles in your AWS account. Navigate to the IAM console and check if there are any roles or policies that grant AWS SRT access to your account. The role should be "AWSShieldDRTAccessRole" and the policy should be "AWSShieldDRTAccessPolicy". If these roles and policies exist and are attached to the SRT, then the SRT has access to your AWS account.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. First, you need to install and configure AWS CLI on your local machine. You can download it from the official AWS website and configure it using the "aws configure" command. You will need to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. Once the AWS CLI is set up, you can use the "aws shield describe-protection" command to list the details of the AWS Shield protections for your account. This command will return a JSON output with the details of the protections.
+
+   Command: 
+   ```
+   aws shield describe-protection --protection-id <your-protection-id>
+   ```
+   Replace `<your-protection-id>` with the ID of the protection you want to check.
+
+3. To check if the Shield Response Team (SRT) has access to your AWS account, you can use the "aws shield describe-assistance" command. This command will return a JSON output with the details of the AWS Shield assistance for your account.
+
+   Command: 
+   ```
+   aws shield describe-assistance --region <your-region>
+   ```
+   Replace `<your-region>` with the region where your AWS Shield is configured.
+
+4. If the SRT has access to your AWS account, the "aws shield describe-assistance" command will return a JSON output with the "emergencyAssistancePlan" field set to "ACTIVE". If the SRT does not have access to your AWS account, the "emergencyAssistancePlan" field will be set to "INACTIVE".
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install and configure AWS SDK for Python (Boto3):
+   First, you need to install and configure AWS SDK for Python (Boto3). You can install it using pip:
+   ```
+   pip install boto3
+   ```
+   Then, configure your AWS credentials. You can configure your credentials either by setting the following environment variables:
+   ```
+   AWS_ACCESS_KEY_ID = 'your_access_key'
+   AWS_SECRET_ACCESS_KEY = 'your_secret_key'
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>